concourse/steps/update_component_deps.mako

```diff
@@ -102,6 +111,16 @@ for from_ref, to_version in determine_upgrade_prs(
     upgrade_pull_requests=upgrade_pull_requests,
     ctx_repo_base_url='${ctx_repo_base_url}',
 ):
+    configured_merge_policy = [
+        policy for policy, filter_func in merge_policies.items() if filter_func(from_ref)
+    ]
+    if len(configured_merge_policy) > 1:
+        raise RuntimeError(f'More than one merge policy applies to {from_ref.name}')
```

Comment:
I think we should be more forgiving here. how about only considering it an error if there is a conflict (two different policies for the same component..)?